src/components/ui-shell/_product-switcher.scss

```diff
@@ -133,34 +133,34 @@
     color: $shell-header-text-02;
   }
 
-  .#{$prefix}--overflow-menu {
-    display: none;
-    justify-content: center;
-    align-items: center;
-    width: mini-units(5);
-  }
-
-  .#{$prefix}--overflow-menu > svg {
-    fill: $shell-header-text-02;
-  }
-
-  .#{$prefix}--overflow-menu:hover {
-    background: $shell-header-bg-04;
-  }
-
-  .#{$prefix}--overflow-menu:hover > svg {
-    fill: $shell-header-text-02;
-  }
-
-  .#{$prefix}--overflow-menu:focus {
-    display: flex;
-    outline: none;
-    box-shadow: inset 0 0 0 3px $shell-header-link;
-  }
-
-  .#{$prefix}--overflow-menu-options__option:hover {
-    background: $shell-header-bg-03;
-  }
+  // .#{$prefix}--overflow-menu {
```

Comment:
@aledavila It seems that these styles were overwriting the default overflow menu styles, so I've commented them out until I could prefix them. I wasn't sure where they were being used, do you know what I can add before this selector to ensure they are only hitting overflow menus inside product switcher?